src/modules/balance/handlers/handlerBalance.mixin.js

```diff
@@ -0,0 +1,75 @@
+import { getBalanceHistory } from '@/apollo/queries/wallets/wallets.graphql';
+import utils from 'web3-utils';
+import BigNumber from 'bignumber.js';
+import { Toast, ERROR } from '@/modules/toast/handler/handlerToast';
+
+export default {
+  name: 'HandlerBalance',
+  data() {
+    return {
+      getTimeseriesData: '',
+      nextKey: ''
+    };
+  },
+  apollo: {
+    /**
+     * Apollo query to return balance history
+     */
+    getTimeseriesData: {
+      query: getBalanceHistory,
+      variables() {
+        return {
+          timeString: Math.floor(this.timeString / 1000),
+          todaysDate: Math.floor(new Date().getTime() / 1000),
+          key: `ACCOUNT_BALANCE_PREFIX_AVG-0xETH-${this.address}`,
+          scale: this.scale,
+          _nextKey: this.nextKey
+        };
+      },
+      skip() {
+        const acceptableScales = ['seconds', 'minutes', 'hours', 'days'];
+        return !acceptableScales.includes(this.scale);
+      },
```

Comment:
Please add  update: data => data.getTimeseriesData, Then in result you can get 
`const nextKey = data.getTimeseriesData.nextKey` as `const nextKey = data.nextKey`

Reply:
done!